modules/storages/app/views/storages/admin/storages/update.turbo_stream.erb

```diff
@@ -0,0 +1,6 @@
+<%= turbo_stream.update ActionView::RecordIdentifier.dom_id(@storage, :storage_view_general_info) do %>
+  <%= render(::Storages::Admin::StorageViewComponent.new(@storage)) %>
```

Comment:
It seems rendering complete `StorageViewComponent` renders- replaces it within general info frame

![Screenshot 2023-10-30 at 1 12 45 PM](https://github.com/opf/openproject/assets/17295175/1ae4ffab-9ca2-4611-889b-3b55d21219cd)

Seems we'll need to extract the general info into a component

Reply:
I'm already on to fixing that.